internal/executor/service/stuck_pod_detector_test.go

```diff
@@ -0,0 +1,261 @@
+package service
+
+import (
+	"testing"
+	"time"
+
+	"github.com/G-Research/armada/internal/common"
```

Comment:
This import block and the next one shoudl swap places ("standatd library", "third-party", "internal").